red hat/tsn-testbench.spec

```diff
@@ -0,0 +1,68 @@
+# SPDX-License-Identifier: BSD-2-Clause
+
+Name: TSN-Testbench
+Version: 5.0
+Release: 1%{?dist}
+Summary: Time-Sensitive Networks Testbench is a set of tools for validating the networks with different protocols
+License: BSD-2
+%undefine _disable_source_fetch
+URL: https://github.com/Linutronix/TSN-Testbench
+Source0: https://github.com/Linutronix/TSN-Testbench/archive/refs/tags/v%{version}.tar.gz
+
+BuildRequires: clang
+BuildRequires: cmake
+BuildRequires: libxdp-devel
+BuildRequires: libyaml-devel
+BuildRequires: llvm
+BuildRequires: mosquitto-devel
+BuildRequires: openssl-devel
+BuildRequires: python3-sphinx
+BuildRequires: python3-sphinx_rtd_theme
+BuildRequires: texinfo
+
+%description
+The TSN Testbench is a real-time and non-real-time traffic validation tool for converged TSN networks. PROFINET as well as OPC/UA PubSub and other configurable protocols are supported.
+
+%prep
+cd %{_topdir}/BUILD
+rm -rf %{name}-%{version}
+git clone https://github.com/Linutronix/TSN-Testbench %{name}-%{version}
+cd %{name}-%{version}
+git checkout v%{version}
+
+%build
+cd %{_topdir}/BUILD/%{name}-%{version}
+cmake -DCMAKE_BUILD_TYPE=Release
+make DESTDIR=${RPM_BUILD_ROOT} install
+
+# Build documentation
+make -C Documentation   man
```

Comment:
I don't think there's a man page yet?

Reply:
~~~
looking for now-outdated files... none found
pickling environment... done
checking consistency... done
writing... linuxtsntestbench.1 { introduction quickstart contribution build usage configuration mqtt test debug qa } done
build succeeded.

The manual pages are in _build/man.
~~~
![image](https://github.com/user-attachments/assets/4cade6de-f78e-4814-a94d-d83614594d7d)

and result

![image](https://github.com/user-attachments/assets/4ed1daba-78c0-46c3-963d-c6a235e5dcdf)